generated/_task_modules/channels/ai_channel.py

```diff
@@ -0,0 +1,202 @@
+from nidaqmx._task_modules.channels.channel import Channel
+from nidaqmx.utils import unflatten_channel_string
+
+
+class AIChannel(Channel):
+    """
+    Represents one or more analog input virtual channels and their properties.
+    """
+    __slots__ = []
+
+    def __repr__(self):
+        return 'AIChannel(name={0})'.format(self._name)
+
+
+    @property
+    def ai_atten(self):
+        """
+        Specifies the amount of attenuation to use.
+        """
+        val = ctypes.c_double()
+
+        cfunc = lib_importer.windll.DAQmxGetAIAtten
+        if cfunc.argtypes is None:
+            with cfunc.arglock:
+                if cfunc.argtypes is None:
+                    cfunc.argtypes = [
+                        lib_importer.task_handle, ctypes_byte_str,
+                        ctypes.POINTER(ctypes.c_double)]
+
+        error_code = cfunc(
+            self._handle, self._name, ctypes.byref(val))
+        check_for_error(error_code)
+
+        return val.value
+
+    @ai_atten.setter
+    def ai_atten(self, val):
+        cfunc = lib_importer.windll.DAQmxSetAIAtten
+        if cfunc.argtypes is None:
+            with cfunc.arglock:
+                if cfunc.argtypes is None:
+                    cfunc.argtypes = [
+                        lib_importer.task_handle, ctypes_byte_str,
+                        ctypes.c_double]
+
+        error_code = cfunc(
+            self._handle, self._name, val)
+        check_for_error(error_code)
+
+    @ai_atten.deleter
+    def ai_atten(self):
+        cfunc = lib_importer.windll.DAQmxResetAIAtten
+        if cfunc.argtypes is None:
+            with cfunc.arglock:
+                if cfunc.argtypes is None:
+                    cfunc.argtypes = [
+                        lib_importer.task_handle, ctypes_byte_str]
+
+        error_code = cfunc(
+            self._handle, self._name)
+        check_for_error(error_code)
+
+
+    @property
+    def ai_raw_samp_justification(self):
+        """
+        Indicates the justification of a raw sample from the device.
+        """
+        val = ctypes.c_int()
+
+        cfunc = lib_importer.windll.DAQmxGetAIRawSampJustification
+        if cfunc.argtypes is None:
+            with cfunc.arglock:
+                if cfunc.argtypes is None:
+                    cfunc.argtypes = [
+                        lib_importer.task_handle, ctypes_byte_str,
+                        ctypes.POINTER(ctypes.c_int)]
+
+        error_code = cfunc(
+            self._handle, self._name, ctypes.byref(val))
+        check_for_error(error_code)
+
+        return int(val.value)
+
+
+    @property
+    def ai_bridge_poly_forward_coeff(self):
+        """
+        Specifies an list of coefficients for the polynomial that
+            converts electrical values to physical values. Each element
+            of the list corresponds to a term othe equation. For
+            example, if index three of the list is 9,the fourth term of
+            the equation is 9x^3.
```

Comment:
Part of this description is cut off.
- "term of the equation" changed to "term othe equation".
- "is 9, the" is missing a space.

![image](https://user-images.githubusercontent.com/17914399/219503607-44ff60a4-d28f-4d4b-bce4-d340e3c593fa.png)

Reply:
Since the metadata was handwritten, the typos in metadata led to this difference. I have updated the metadata accordingly.